startlxqtwayland.in

```diff
@@ -95,11 +95,11 @@ if [ $COMPOSITOR = "labwc" ]; then
     fi
 
 # enable cursor on VM
-    VIRT=$(systemd-detect-virt)
-    if [ ! "$VIRT" = none ]; then
+    if [ $(hostnamectl | grep Virtualization) ];then
```

Comment:
Despite the commit log message, here you're using systemd's component

Reply:
> What about using cat /sys/devices/virtual/dmi/id/product_family ?

I don't think it's reliable for this kind of usage.

> We could also drop this condition

Why dropping it totally? Why not dropping it only in the exceptional case where `systemd` doesn't exist?

I don't think there's a tool as handy as `systemd-detect-virt`.